UIKitDz/UIKitDz/FifthViewController.swift

```diff
@@ -0,0 +1,141 @@
+//
+//  FifthViewController.swift
+//  UIKitDz
+//
+//  Created by Admin on 01.07.2021.
+//
+
+import UIKit
+//5. Создать экран оплаты, на котором необходимо
+//спрятать Navigation Bar, а вместо него добавить
+//заголовок оплата, отображаем пиццу и выбранные
+//ингридиенты с предыдущего экрана , добавить выбор
+//способа оплаты (если выбираем наличные отключается
+//свитч оплата картой и наборот)
+//6. При нажатии на кнопку оплаты показываем алерт об
+//успешной оплате заказа, при нажатии на кнопку ОК мы
+//переходим на экран выбора еды (второй после экрана
+//авторизации)
+
+class FifthViewController: UIViewController {
```

Comment:
```suggestion
final class FifthViewController: UIViewController {
```
и во всех классах ниже, от которых нет наследования